src/style.css

```diff
@@ -0,0 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+:root {
+  ---accent-page-color: #00acdc;
+}
+
+/* #region reset */
+body,
+p,
+h1 {
+  margin: 0;
+}
+/* #endregion */
+
+.product-card {
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+
+  width: 200px;
+}
+
+.product-card__image {
+  margin-top: 16px;
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__name {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  margin-top: 4px;
+  align-self: flex-start;
+
+  color: #616070;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card-rate {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: 16px;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars-star {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: auto;
+}
+
+.stars-star:first-child {
+  margin-left: 0;
+}
+
+.stars--4 .stars-star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+.product-card-rate__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.price-container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  margin-top: 24px;
+
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price-value {
+  color: #060b35;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.product-card-button {
+  display: flex;
+  width: 100%;
+  padding-top: 16px;
+}
+
+.buy-button {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  height: 40px;
+  border: none;
+  border-radius: 5px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+
+  color: white;
+  background-color: var(---accent-page-color);
+
+  cursor: pointer;
+}
+
+.buy-button:hover {
+  color: var(---accent-page-color);
+
+  background-color: white;
+  border: 1px solid var(---accent-page-color);
+  border-radius: 5px;
```

Comment:
here you don't need  `border-radius` it is in the button class and it doesn't change